src/components/ApyCalculatorModal/index.tsx

```diff
@@ -159,10 +159,10 @@ const ApyCalculatorModal: React.FC<ApyCalculatorModalProps> = ({
         <Text fontSize="12px" color="textSubtle">
           {t(
             `Calculated based on current rates. Compounding %freq%x daily. Rates are estimates provided for your convenience only, and by no means represent guaranteed returns.`,
-            { fee: compoundFrequency.toLocaleString() },
+            { freq: compoundFrequency.toLocaleString() },
           )}
         </Text>
-        {performanceFee && (
+        {performanceFee > 0 && (
```

Comment:
Default set to 0 for this value in the component props (for regular pools that don't have a performance fee).  https://github.com/pancakeswap/pancake-frontend/pull/1099/files#diff-f813c61bd447501445cdeefeef46936f5ca3c2d8c2adb7bd65c088bf9870c0c9R39

TBH not really sure why it's rendering 0 after this check 🤷 

```
const performanceFee = 0
performanceFee && doSomething()
// doSomething fires
```